### ISSUE TITLE
Is there a limit to the depth?

### ISSUE THREAD

**Issue by Geoffrey-Guest** [timelyportfolio/sunburstR#58]
Really nice package! I've been applying it to structural path analysis in environmental life cycle assessment. I have been running into an error though. When I select a depth of greater than 12 the sunburst that's rendered leaves a gap. Are you aware of any limitation in terms of # of layers or any other constraint that might be causing the rendered sunburst to leave a missing piece of the pie?

**Comment by cjyetman** [comment]
Could you provide a [reprex](http://reprex.tidyverse.org/articles/reprex.html)? That would likely facilitate getting it fixed faster.

**Comment by timelyportfolio** [comment]
reprex are always helpful, but I'll guess that these [lines](https://github.com/timelyportfolio/sunburstR/blob/master/javascript/src/sunburst-chart.js#L157-L161) are the culprit since you are > 12 layers deep.  What approximate sizes are the missing slices?

**Comment by Geoffrey-Guest** [comment]
Actually my apologies, as I look more closely at my code I'm the one at fault here. I had a labelling issue that didn't generate the final label in the list of pathways under certain conditions. Thanks for highlighting those lines though!

Another issue I could bring up is to do with whether only 1 layer is allowed. When I tried it didn't work and I required at least 2 layers. I then ensured I always had 2 layers and so I didn't spend much time inspecing it.
